docs/developer-guide/views-examples.md

```diff
@@ -0,0 +1,120 @@
+---
+description: Learn about the most used custom view examples of Lite XL.
+---
+
+## Toolbar
+
+### Tutorial
+
+```lua
+-- mod-version:3 -- lite-xl 2.1
+```
+
+This is required to specify Lite XL version compatibility.
+
+The instructions that begin with `require` add necessary libraries.
+
+```lua
+local toolbar = common.merge({
+  -- plugin configuration options go here
+}, config.plugins.build)
+```
+
+The code above contains the (optional) configuration options of the plugin.
+
+The `get_plugin_directory()` function gets the plugin folder path and tells the 
+plugin where to find the icon font.
+The iteration code is used to check if every folder detected by `get_plugin_directory()` 
+actually exists and if the folder doesn't exist, a `nil` value is returned.
+
+`local ToolBar = ToolbarView:extend()` is used to create a new custom View and make it 
+inherit ToolbarView.
+
+```lua
+function ToolBar:new()
+  ToolBar.super.new(self)
+  self.toolbar_font = renderer.font.load(get_plugin_directory() .. PATHSEP .. "toolbar.ttf", style.icon_big_font:get_size())
+  self.toolbar_commands = {
+    {symbol = "", command = "core:open-log"},
```

Comment:
Can you make these symbol visible, for example with escape codes? I think this isn't needed in Lua, but it might be confusing if it just rendered as blank spaces.

Reply:
I don't understand what the problem is here.

Reply:
Right now the symbol string shows a box with a question mark. I'm not sure if it's possible to put a unicode literal like `\u1FFF` (pretty sure there is a function that converts a number to a unicode codepoint) instead of the string itself.

![image](https://github.com/takase1121/lite-xl-docs/assets/20792268/d692bc63-ab60-4c1d-acce-5232a1531e2c)

Reply:
I could just replace those custom glyphs with conventional letters, just like Adam did for his `build` plugin's toolbar.

Reply:
I outright replaced the symbols with letters (`A`, `B`, ...), like how the default Lite XL toolbar does.